ext/standard/filestat.c

```diff
@@ -707,6 +711,37 @@ PHP_FUNCTION(touch)
 /* }}} */
 #endif
 
+#if HAVE_GETFSENT
+PHP_FUNCTION(fstab)
+{
+	struct fstab *entry;
+	zend_long i;
+
```

Comment:
If a function doesn't accept any arguments, this should be checked with `zend_parse_parameters_none()`, i,e.
https://github.com/php/php-src/blob/17fb7241e458032146851d538221a32e30a67909/Zend/zend_builtin_functions.c#L339-L341

Reply:
Good to know, knew the other macros only.